doc/empty-prompt.txt

```diff
@@ -0,0 +1,77 @@
+*empty-prompt.txt*		Add mappings only when shell prompt is empty
+
+Author: tyru <tyru.exe@gmail.com>
+Licence: GPL-3 or any later versions
```

Comment:
Please ask me which Licence you'd like, since the current licence is AS-IS.

Reply:
Please change to MIT License.
Because I referenced kana/vim-submode.and it is MIT License.
https://github.com/tyru/empty-prompt.vim/blob/8e70208defd3ef8ba6da87f81ce8c46542fba2ea/autoload/empty_prompt.vim#L38

Reply:
Done! 67caa387a88a49accad8a73257a332396ae7512d